src/js/button.js

```diff
@@ -78,6 +83,8 @@ vjs.Button.prototype.onKeyPress = function(event){
 };
 
 // Blur - Remove keyboard triggers
-vjs.Button.prototype.onBlur = function(){
-  vjs.off(document, 'keydown', vjs.bind(this, this.onKeyPress));
+Button.prototype.onBlur = function(){
+  Events.off(document, 'keydown', Lib.bind(this, this.onKeyPress));
 };
+
+export default Button;
```

Comment:
Yeah, it seems like there's a pattern emerging (among ES6 modules at least) where imports are at the top and exports are at the bottom. I like that myself, at least for consistency.